pkg/provider/azure_storageaccount.go

```diff
@@ -37,7 +37,7 @@ import (
 const SkipMatchingTag = "skip-matching"
 const LocationGlobal = "global"
 const GroupIDFile = "file"
-const PrivateDNSZoneName = "privatelink.file.core.windows.net"
+const PrivateDNSZonePrefix = "privatelink.file."
```

Comment:
nit: The variable doesn't need to be exported.

Reply:
fixed, could you approve, thanks.